clang/test/Analysis/builtin_bitcast.cpp

```diff
@@ -30,3 +30,24 @@ void test(int i) {
   clang_analyzer_dump(g4);
   // expected-warning@-1 {{&i [as 64 bit integer]}}
 }
+
+struct A {
+  int n;
+  void set(int x) {
+    n = x;
+  }
+};
+using ptr_size = decltype(sizeof(void *));
```

Comment:
Do I see it correctly that `ptr_size` just an alias of `size_t`? If yes, then why don't you use the standard name?

Reply:
Fixed. Now using `size_t`.

Reply:
Thanks!